environment/task_admin_account_cleanup.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_cloudwatch_event_rule" "monday-night" {
+  name                = "monday-night-${local.environment}"
+  description         = "Monday Night scheduled tasks"
+  schedule_expression = "cron(0 3 ? * 1 *)"
```

Comment:
Set it to run every Monday am, this way we can fix any erroneous deletions if they occur.